dartagnan/src/main/java/com/dat3m/dartagnan/GlobalSettings.java

```diff
@@ -1,52 +1,22 @@
 package com.dat3m.dartagnan;
 
-import com.dat3m.dartagnan.solver.caat4wmm.coreReasoning.CoreReasoner;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.sosy_lab.common.configuration.Configuration;
-import org.sosy_lab.common.configuration.InvalidConfigurationException;
-import org.sosy_lab.common.configuration.Options;
 
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-@Options
 public class GlobalSettings {
```

Comment:
What about moving the static methods to utils/Utils.java and getting rid of this class?

Reply:
I don't think I'm a fan of that. The content is not some random utility but indeed a sort of global configuration/setting. For example, if we ever move the `cat` or `output` directories (or rename them), we would need to update it in this class. I think a class that contains (all) functionality/information related to our environment variables (`DAT3M_HOME/DAT3M_OUTPUT`) and internal project structure should not be in some obscure utility location.
I would rather suggest to move the content to the main class `Dartagnan` (or not move it at all).

Reply:
The lets simply keep it as it is.